examples/blog-with-comment/pages/_document.tsx

```diff
@@ -1,8 +1,14 @@
-import NextDocument, { Html, Head, Main, NextScript } from 'next/document'
```

Comment:
I think we can get rid of this file. `meta.title` is overridden in `_app` already to comply with https://github.com/vercel/next.js/blob/canary/errors/no-document-title.md

Let's remove it from this file and get rid of the `meta` object (just inline the `description` maybe)

I first thought we could remove the file entirely but there is some basic styling applied. Adding it back elsewhere would require creating a file in a different location, so there is no removed complexity.

We can turn this into a functional component though. See https://nextjs.org/docs/advanced-features/custom-document